### PR TITLE
More copy edits

### PR DESCRIPTION
**Pull request by WillGardella**
Removed German style capitalization, other copy edits.

### CODE REVIEW

File: docs/user/index.md [hunk 1]
```diff
@@ -134,10 +130,10 @@ status.
 
 ### Organization webhooks
 
-GitHub Organization members that have the [Owner role][owners], are allowed
+GitHub organization members that have the [owner role][owners], are allowed
```
Comment (ddgenome) [inline docs/user/index.md]: GitHub refers to this role as "Owner", not "owner". For example, https://github.com/orgs/atomist/people

File: docs/user/index.md [hunk 2]
```diff
@@ -134,10 +130,10 @@ status.
...
 to configure Organization-wide webhooks.  This is convenient
 because it only has to be configured once;
-however, you will require a User who has the `Owner` role in
+however, you will require a user who has the `owner` role in
```
Comment (ddgenome) [inline docs/user/index.md]: "Owner"

File: docs/user/index.md [hunk 3]
```diff
@@ -152,28 +148,28 @@ GitHub organization.
 ![GitHub Authorize Organization Webhook](img/authorize-org-hook.png)
 
 Since you might be a member of many GitHub organizations, Atomist may ask you to
-choose which Organization you are enrolling.
+choose which organization you are enrolling.
 
 ![Choose GitHub Organization](img/choose-org.png)
 
-Finally, you will be presented with a button to configure the Org-level webhook.
+Finally, you will be presented with a button to configure the org-level webhook.
```
Comment (ddgenome) [inline docs/user/index.md]: "org-level" is an internal term, we should probably use "organization webhook"

File: docs/user/index.md [hunk 4]
```diff
@@ -242,8 +239,8 @@ def notifyAtomist(buildStatus, buildPhase="FINALIZED",
 }
 ```
 
-Then call `notifyAtomist` when the build starts, e.g., in the first
-stage, and ends, i.e., in the `post` block, sending the appropriate
+Then call `notifyAtomist` when the build starts (here, in the first
+stage) and ends ( in the `post` block), sending the appropriate
```
Comment (ddgenome) [inline docs/user/index.md]: "here" isn't right. If you are going to get rid of "e.g.", I'd just remove it.

There appears to be a space between the open parenthesis and "in the `post` block".

New text:  "... when the build starts (in the first stage) and ends (in the `post` block), ..."

**Comment by WillGardella** [comment]
In case it's not clear above, I changed the YAML to name the Creating Projects page and moved it.

**Comment by ddgenome** [comment]
Saw that, cool.

File: docs/user/index.md [hunk 5]
```diff
@@ -54,23 +54,21 @@ Currently the authorization process asks you to authorize two things:
...
+    member.
 2.  Atomist requests a scope called "Modify public channels".
-    This scope allows Atomist to help you setup channels to bring
-    information from external systems into Slack.  For example, when
-    you create a project in a new GitHub repo, Atomist can help you
-    direct events from that project into a new channel that only
-    people working on that project decide to join.  The Atomist app
+    This scope allows Atomist to help you setup channels.  For example, 
+    when you create a project in a new GitHub repo, Atomist can create 
+    a Slack channel to go with it.
```
Comment (claymccoy) [inline docs/user/index.md]: way better!

File: docs/user/index.md [hunk 6]
```diff
@@ -152,28 +148,28 @@ GitHub organization.
...
 
 ```
 you> @atomist install webhook
 ```
 
-The bot will now ask for the `Owner` of the Repository.  This question will be
-skipped if there is only valid choice (your User account).  Second the bot will
-ask you to select the Repository to recieve the new webhook.
+The bot now asks for the `owner` of the repository.  This question is
+skipped if there is only valid choice (your user account).  Next the bot
```
Comment (claymccoy) [inline docs/user/index.md]: if there is only ONE valid choice

File: mkdocs.yml [hunk 7]
```diff
@@ -62,15 +62,15 @@ markdown_extensions:
 pages:
   - Introduction: index.md
   - Using Atomist: user/index.md
+  - Quick Start: developer/quick-start.md
```
Comment (claymccoy) [inline mkdocs.yml]: Okay, no need to change directory structure and links. Works for me.

File: docs/user/index.md (hunk 2 above)
Comment (ddgenome) [reply docs/user/index.md]: Gave the thumbs up, yet it remains lower case. I see how it is.

**Pull request merged**